app/assets/stylesheets/base/grid.scss

```diff
@@ -0,0 +1,11 @@
+@import '../vars/variables';
+
+// Gap
+@include generate-classes(
```

Comment:
I think we're good to use CSS grid now since we do not support IE. Thoughts @benhalpern?

Reply:
Btw. we're already using it in some places, like Listings without any fallback to flex.